src/openMVG/features/svgVisualization.cpp

```diff
@@ -15,7 +15,7 @@
 #include <algorithm> 
```

Comment:
the header i was referring to is the comment at the very beginning of the file, the stuff with author date etc